### ISSUE TITLE
Something wrong when finetuning flux-kontext with lora

### ISSUE THREAD

**Issue by Mr-ind1fferent** [bghira/SimpleTuner#1711]
multidatabackend.json

[
    {
        "id": "edited-images",
        "disabled": false,
        "type": "local",
        "instance_data_dir": "/home/bobo/project/SimpleTuner/dataset/gt_images",
        "minimum_image_size": 1024,
        "maximum_image_size": 1536,
        "target_downsample_size": 1024,
        "resolution": 1024,
        "resolution_type": "pixel_area",
        "caption_strategy": "textfile",
        "cache_dir_vae": "cache/vae/flux-edit",
        "vae_cache_clear_each_epoch": false,
        "conditioning_data": ["reference-images"]
    },
    {
        "id": "reference-images",
        "disabled": false,
        "type": "local",
        "instance_data_dir": "/home/bobo/project/SimpleTuner/dataset/rgb_images",
        "minimum_image_size": 1024,
        "maximum_image_size": 1536,
        "target_downsample_size": 1024,
        "resolution": 1024,
        "resolution_type": "pixel_area",
        "caption_strategy": null,
        "cache_dir_vae": "cache/vae/flux-ref",
        "vae_cache_clear_each_epoch": false,
        "conditioning_type": "reference_strict"
    },
    
    {
        "id": "text-embed-cache",
        "dataset_type": "text_embeds",
        "default": true,
        "type": "local",
        "cache_dir": "cache/text/flux"
    }
]

config.json

{
    "aspect_bucket_rounding": 2,
    "base_model_precision": "int8-quanto",
    "caption_dropout_probability": 0.1,
    "checkpointing_steps": 50,
    "checkpoints_total_limit": 5,
    "conditioning_multidataset_sampling": "random",
    "data_backend_config": "/home/bobo/project/SimpleTuner/config/multidatabackend.json",
    "disable_benchmark": false,
    "pretrained_model_name_or_path": "/home/bobo/project/SimpleTuner/model/kontext",
    "eval_dataset_id": "edited-images",
    "fuse_qkv_projections": false,
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": true,
    "gradient_checkpointing_interval": 1,
    "hub_model_id": "flux-kontext-test-lora",
    "learning_rate": 1e-8,
    "lora_rank": 16,
    "lora_type": "standard",
    "lr_scheduler": "constant",
    "lr_warmup_steps": 100,
    "max_train_steps": 100,
    "minimum_image_size": 0,
    "mixed_precision": "bf16",
    "model_family": "flux",
    "model_flavour": "kontext",
    "model_type": "lora",
    "num_eval_images": 4,
    "num_train_epochs": 0,
    "optimizer": "optimi-lion",
    "output_dir": "output/examples/kontext.peft-lora-waymo",
    "push_checkpoints_to_hub": false,
    "push_to_hub": false,
    "quantize_via": "cpu",
    "report_to": "none",
    "resolution": 512,
    "resolution_type": "pixel_area",
    "resume_from_checkpoint": "latest",
    "seed": 42,
    "train_batch_size": 1,
    "use_ema": false,
    "vae_batch_size": 6,
    "validation_guidance": 2.5,
    "validation_guidance_rescale": 0.0,
    "validation_num_inference_steps": 25,
    "validation_prompt": "remove degradation",
    "validation_resolution": "1024x1024",
    "validation_seed": 42,
    "validation_steps": 50,
    "validation_torch_compile": false
}

but something wrong when training 

<img width="878" height="423" alt="Image" src="https://github.com/user-attachments/assets/be0c95a7-0b9b-43dc-93b4-e91c96932d8e" />

and this is my output

<img width="326" height="549" alt="Image" src="https://github.com/user-attachments/assets/d95394e4-ecac-453b-be5d-b0409241570d" />

Wondering how to solve the problem, is there anything wrong with my json file

**Comment by bghira** [comment]
remove eval_dataset_id setting.
